dicomImport/matRad_importDicomRtss.m

```diff
@@ -66,9 +66,11 @@
                              
     structures(i).structNumber = structInfo.ROIContourSequence.(...
                                  listOfContStructs{i}).ReferencedROINumber;
-    structures(i).structColor  = structInfo.ROIContourSequence.(...
-                                 listOfContStructs{i}).ROIDisplayColor;  
-                             
+    if isfield(structInfo.ROIContourSequence.(listOfContStructs{i}),'ROIDisplayColor')
+        structures(i).structColor  = structInfo.ROIContourSequence.(...
+                                     listOfContStructs{i}).ROIDisplayColor;  
+    end
+
```

Comment:
Non-defined structure color can indeed happen. I noticed this first for a point structure, but I am not completely sure if a volumetric structure has to have a defined color.
I did not check with the GUI functions if actually use the structure color anywhere and if we therefore need a default value.